src/core/expression/qgsexpressioncontextutils.cpp

```diff
@@ -975,3 +975,99 @@ QgsScopedExpressionFunction *QgsExpressionContextUtils::GetLayerVisibility::clon
   func->mScaleBasedVisibilityDetails = mScaleBasedVisibilityDetails;
   return func;
 }
+
+//
+// mesh expression context
+//
+
+/// @cond PRIVATE
+class CurrentVertexZValueExpressionFunction: public QgsScopedExpressionFunction
+{
+  public:
+    CurrentVertexZValueExpressionFunction():
+      QgsScopedExpressionFunction( "$vertex_z",
+                                   0,
+                                   QObject::tr( "Meshes" ),
```

Comment:
I think the group name should not be translatable - that's how it is with other functions too